_posts/2025-05-21-geospatial-techno.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Nima Ghasemloo
+layout: post
+title: GeoServer Installation and Upgrade Guide
+date: 2025-05-21
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### GeoServer Installation and Upgrade Guide
+In this session, we will install GeoServer on Windows using the Web Archive installation method and upgrade to a new version, while retaining existing data.
+
+If you want to access the complete tutorial, click on the [link](https://www.youtube.com/watch?v=SpTdPIRxjU0&list=PL_ITaxp1Ob4sjk24Stboa5XbO0LGdEKbL).
+
+[![](https://img.youtube.com/vi/SpTdPIRxjU0/0.jpg)](https://www.youtube.com/watch?v=SpTdPIRxjU0&list=PL_ITaxp1Ob4sjk24Stboa5XbO0LGdEKbL)
+
+## Introduction
+GeoServer is a versatile, Java-based application compatible with various operating systems, provided a suitable Java Virtual Machine (JVM) is available. The latest versions of GeoServer have been tested with both Oracle JRE and OpenJDK.
+
+The GeoServer WAR file is a platform-independent web archive designed for deployment on application servers. Apache Tomcat is the recommended servlet container due to its robust integration capabilities and comprehensive documentation. This setup allows multiple web applications to run concurrently, enabling GeoServer to operate alongside other Java-based services, enhancing server versatility.
+
+**Note.** This guide outlines the installation of GeoServer 2.25.x using Java 17 and Apache Tomcat 9, followed by upgrade instructions. To ensure you have the latest release, please visit this [link](https://geoserver.org/download/) and avoid using older versions of GeoServer.
```

Comment:
2.25 is really no longer supported; I understand that you made the video previously. I am glad you included the note.